configure.ac

```diff
@@ -18,6 +18,7 @@ fi
 
 # Checks for programs.
 AC_PROG_CC
+AC_PROG_CC_STDC
```

Comment:
I see, I removed `AC_PROG_CC_C99` as it was sait it was obsolete, but didn't think readding a replacement was needed.

Reply:
Yes, it obsoleted by newest autotools or compiler working in C99 mode as default.
But for backwards-compatibility with other build system we must left STDC keyword here or remove loop initial declarations such as "for (int i=0; i>x; i++)" to "int i; for (i=0;i>x;i++)" - no code using C99 features in project now, but only loop initial declarations in almost all files.
If we will rewrite it, code will work also with exotic compilers, as will match pure C rules.

Reply:
I'm not a big fan of `for (int i=` as I would rather declare the needed vars at beginning of scope. I would certainly be willing to run a cleanup pass removing all those, not for C99 compatibility but simply because I don't like it :)

Reply:
There is big diffirence:

Reply:
I'm aware of the implications, but a 2nd for loop not initializing the counter would be a pretty weird design pattern.
I did change some of these things during my initial large win32 refactor because some of the vars were shadowing global/outer scope vars. I just didn't replace them all.

Reply:
So if we will decide to remove STDC from autotools, we also must tune compiler flags which will actually disable c99 (as some compilers enable c99 by default, them will compile code which uses c99 features without c99-enabling flag, but others will fail without this flag).